src/sentry/snuba/metrics/extraction.py

```diff
@@ -627,13 +640,29 @@ def apdex_tag_spec(project: Project, argument: Optional[str]) -> List[TagSpec]:
     ]
 
 
+def user_misery_tag_spec(project: Project, argument: Optional[str]) -> List[TagSpec]:
+    """A metric that counts the number of unique users who were frustrated; "frustration" is
+    measured as a response time four times the satisfactory response time threshold (in milliseconds).
+    It highlights transactions that have the highest impact on users."""
+    field, threshold = _get_field_and_threshold(project, argument)
+
+    return [
```

Comment:
This is likely nowhere near what we want since we're looking for something like this:

(count_miserable(user, **threshold_value**) + 5.8875) / (count_unique(user) + 5.8875 + 111.8625) = **The score we want**
https://github.com/getsentry/sentry/blob/b29efaef31605e2e2247128de0922e8dca576a22/src/sentry/search/events/datasets/discover.py#L206-L230